### ISSUE TITLE
Problem with running scripts that require 'jtsk2wgs84'

### ISSUE THREAD

**Issue by Ades551** [maral/text-to-map#1]
I had an issue with running `npm run address-points`. I was getting an error

```
import jtsk2wgs84 from './src/jtsk2wgs84'
^^^^^^

SyntaxError: Cannot use import statement outside a module
```

I fixed it by adding `"type": "module"` into the `@arodax/jtsk2wgs84/package.json`. It will be good to mention it in README or maybe it can be somehow fixed within your project (don't know).

**Comment by maral** [comment]
Thank you for the feedback! Honestly, I am really confused and tired of this kind of error, those happened to me about 5 times at different points of the development and each took me quite some time to crack - yet another flag or config change to add... I will try to look into it and fix it ASAP.

**Comment by maral** [comment]
Fixed by 3a2206ae7684b814e5341b496a533fad0f20a1db.

**Comment by maral** [comment]
Fixed by 3a2206ae7684b814e5341b496a533fad0f20a1db.
